src/query/functions/tests/it/scalars/comparison.rs

```diff
@@ -386,3 +387,21 @@ fn test_regexp(file: &mut impl Write) {
     run_ast(file, "lhs regexp rhs", &columns);
     run_ast(file, "lhs rlike rhs", &columns);
 }
+
+fn test_sounds_like(file: &mut impl Write) {
```

Comment:
could be removed.

You can add sqllogic tests to test `sounds like`.

Reply:
Is this test unnecessary? This it test is for AST sytax check . can we just keep it?